universal-application-tool-0.0.1/app/models/Version.java

```diff
@@ -76,4 +82,36 @@ public Instant getSubmitTime() {
         .filter(q -> q.getQuestionDefinition().getName().equals(name))
         .findAny();
   }
+
+  public ImmutableList<String> getTombstonedProgramNames() {
```

Comment:
Suggestion (blocking): please add a comment explaining what it means for a question or program to have its name in the tombstoned list.

Reply:
Done.